### PR TITLE
feat: whitepaper abstract and intro

### PR DESCRIPTION
**Pull request by jorgeantonio21**

### CODE REVIEW

File: papers/whitepaper.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (joehill7) [inline papers/whitepaper.md:77]: The Atoma Network will be powered by its TOMA token. Through robust tokenomics, compute providers, protocol participants, and developers will be rewarded by their contribution to maintaining the network longevity. The TOMA token will be required to pay for AI inference fees, for nodes to deposit collateral to be able to participate on the network, and also for developers to be paid for their contributions to the Atoma Network.

This growth (of the network?) incentivizes node operators to acquire better compute hardware, and model creators to develop new AI models specialized for applications such as financial market predictions. It also incentivizes developers to build applications that drive demand for the available compute resources on the Atoma Network. 

The tokenomics follow an initial period of inflation in order to subsidize the network's growth. After the network's inflationary period (which will last a few years), the tokenomics will follow a more balanced dynamics, through tail emissions and burning mechanisms.

Together, both mechanisms will lead to a gradual decrease in the total supply of the TOMA token. TOMA will increase in value provided the demand for Atoma Network increases, while also allowing for some new allocation of the token to the protocol's treasury and to value providers (in the form of new node providers, model creators, developers, etc)

File: papers/whitepaper.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+In order to empower the next generation applications in the intersection of Web3 and AI, both data and model privacy is of utmost importance. Through the use of TEEs, we will not only be able to provide verifiable AI compute, but also provide full data privacy to the end user. This means that Atoma nodes will not be able to access any sensitive data, such as personal information, while also not being able to access the output generated by the AI model. Only the end user will be able to decrypt the generated output, and can have high assurance that the input to the model was not shared with any third party. Through this, users can have full control of their data, allowing for AI models to operate on sensitive data. Privacy will unlock the full potential of AI, allowing for the creation of new data economic revenue models, as well applications that can operate on the user's behalf without revealing anything about the user. 
+
```
Comment (joehill7) [inline papers/whitepaper.md:79]: In order to empower the next generation of applications in the intersection of Web3 and AI, both data and model privacy is of the utmost importance. Through the use of TEEs, we will not only be able to provide verifiable AI compute, but also provide full data privacy to the end user. 

This means that Atoma nodes will not be able to access any sensitive data, such as personal information, while also not being able to access the output generated by the AI model. Only the end user will be able to decrypt the generated output, and can have high assurance that the input to the model was not shared with any third party. 

Through this, users can have full control of their data, allowing for AI models to operate on sensitive data. Privacy will unlock the full potential of AI, allowing for the creation of new data centric economic revenue models as well as applications that can operate on the user's behalf without revealing anything about the user.

File: papers/whitepaper.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+On another note, the development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will be crucial to democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work, allowing for experts in the community to earn a continuous stream of revenue through their model utilization.
+
```
Comment (joehill7) [inline papers/whitepaper.md:81]: The development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will be crucial to democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work, allowing for experts in the community to earn a continuous stream of revenue through their model utilization.

File: papers/whitepaper.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+A decentralized platform such as Atoma Network will put forth combined efforts that will ultimately lead to a new era of use cases within decentralized applications, driving financial incentives for computing power to be partially decouple from centralized cloud providers. Smart contracts will not only deploy new financial products, but also be able to deploy and orchestrate compute resources, powering new decentralized protocols.
```
Comment (joehill7) [inline papers/whitepaper.md]: As a decentralized platform, Atoma Network will put forth combined efforts that will ultimately lead to a new era of use cases within decentralized applications through driving financial incentives for computing power to be partially decoupled from centralized cloud providers. Smart contracts will not only deploy new financial products, but also be able to deploy and orchestrate compute resources, powering new decentralized protocols.

**Review by joehill7 — CHANGES_REQUESTED** [review]
Lot of rewording / grammar changes. 

Please review line 38/39. It's slightly confusing as how the token being inflationary helps the growth of the network

File: papers/whitepaper.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+## Jorge Antonio and Hisham Khan
+
+### Abstract
+
+Atoma is a permissionless decentralized protocol specialized for heavy verifiable compute. Atoma derives its features from compute providers, referred to as execution nodes (or simply nodes). Nodes on the Atoma Network are responsible for processing different types of compute requests.
+
+AI inference requests are often processed through nodes that run Large Language models (LLMs) different types of inputs (text, images, video, audio, etc) and produce some form of output (which can be either text, or other formats for multi-modality capabilities). More standard and traditional models for time series predictions, recommendation systems, etc, will also be supported on the Atoma Network.
+
+Due to the recent advances in the field of generative AI, the demand for such compute is expected to grow exponentially in the coming years. It is estimated that decentralized cloud/edge infrastructure is expected to grow substantially by 2027, meaning it is likely that there will be substantial intersections between demand for compute, AI, and decentralized infrastructure.
```
[the remark this answers is not in the document]
Comment (jorgeantonio21) [reply papers/whitepaper.md]: I think I remove this sentence altogether, already.

[the remark this answers is not in the document]
Comment (joehill7) [reply papers/whitepaper.md]: Ok got you

File: papers/whitepaper.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Even though AI models are becoming more powerful at a fast pace, they are also becoming larger requiring higher memory and compute resources, overall. For this reason, to run the best AI models, it is crucial to have enough powerful hardware to run them at scale. For this reason, Atoma has been throughly designed to provide the right financial incentives to compute providers (also referred to as nodes) to acquire and deploy the best hardware. Such incentives combined with a robust efficient inference optimizations will lead to the best possible hardware resource utilization driving demand and lowering costs.
```
[the remark this answers is not in the document]
Comment (joehill7) [reply papers/whitepaper.md]: Repetition of the reference to 2027 here as well, so remove here

[the remark this answers is not in the document]
Comment (jorgeantonio21) [reply papers/whitepaper.md]: I have refactored this already.

File: papers/whitepaper.md (hunk 1 above)
Comment (jorgeantonio21) [reply papers/whitepaper.md:77]: Already refactored

File: papers/whitepaper.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+That said, Bitcoin, Ethereum and other blockchains have shown that decentralized applications can align financial incentives with massive network growth effects. These technologies provide the right framework to align incentives between compute, financial resources and permissionless communities. Inspired by the rise of decentralized applications, we are committed to build the Atoma Network, a permissionless decentralized protocol specialized for heavy verifiable AI compute. The goal of the Atoma Network is to allow compute providers to be able to monetize their compute resources in the most permissionless and transparent way possible, while being exposed to fair market pricing. In order to achieve this, we are in the process of developing Atoma node's infrastructure, which allows any compute provider (also referred to as a node) to easily deploy software that allows to execute AI compute requests, through the Atoma Network. Such infrastructure is highly optimized for best GPU utilization resources, incorporating the most recent advances in LLM memory management techniques (such as Paged Attention and vLLM), etc.
```
[the remark this answers is not in the document]
Comment (jorgeantonio21) [reply papers/whitepaper.md]: Previously refactored.

File: papers/whitepaper.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+This means that Atoma nodes will not be able to access any sensitive data, such as personal information, while also not being able to access the output generated by the AI model. Only the end user will be able to decrypt the generated output, and can have high assurance that the input to the model was not shared with any third party.
+
+Through this, users can have full control of their data, allowing for AI models to operate on sensitive data. Privacy will unlock the full potential of AI, allowing for the creation of new data centric economic revenue models as well as applications that can operate on the user's behalf without revealing anything about the user.
+
+The Atoma Network can be used by Web2 companies and builders, specially software companies that are built from the ground up to be AI-centric, fast growing AI providers (such as Anthropic, etc) and software companies committed to integrating AI into their products (think of Canvas, etc). The reason for these companies to want to operate on the Atoma Network comes from the fact that a network like Atoma is positioned to drive down the costs of GPU utilization.
```
Comment (joehill7) [inline papers/whitepaper.md:68]: The Atoma Network can be used by Web2 companies and builders, especially software companies that are built from the ground up to be AI-centric, i.e fast growing AI providers (such as Anthropic, etc) and software companies committed to integrating AI into their products (think of Canvas, etc). The reason for these companies to operate on the Atoma Network comes from the fact that Atoma is positioned to drive down the costs of GPU utilization.

File: papers/whitepaper.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+Due to the current GPU chips shortage, together with a waste of GPU utilization (meaning less GPU cycles being used for AI inference), hardware providers face sunk costs by not utilize their infrastructure efficiently. Therefore, such hardware providers are able to offer those fractional GPU cycles at a much lower cost compared to other larger cloud providers. Notice, for example, that renting an H100 from AWS today, customers need to commit to a 1-year lease, due to market supply constraints. It is also unlikely that a customer needs to use a GPU at full capacity during the 1-year period. There is furthermore, a high number of idle GPUs in the market, that were used before as mining hardware (e.g., when ETH transitioned from PoW to PoS). Such hardware available can be repurposed for AI inference.
```
Comment (joehill7) [inline papers/whitepaper.md:70]: Due to the current GPU chips shortage, together with a waste of GPU utilization, (less GPU cycles being used for AI inference), hardware providers face sunk costs by not using their infrastructure efficiently. Therefore, hardware providers are able to offer those fractional GPU cycles at a much lower cost compared to other larger cloud providers. 

As an example, a customer that wants to rent an H100 from AWS today needs to commit to a 1-year lease, due to market supply constraints. It is also unlikely that a customer needs to use a GPU at full capacity during the 1-year period. There is high number of idle GPUs in the market that were used before as mining hardware (e.g., when ETH transitioned from PoW to PoS) which could be repurposed for AI inference.

File: papers/whitepaper.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+
+The development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will be crucial to democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work, allowing for experts in the community to earn a continuous stream of revenue through their model utilization.
```
Comment (joehill7) [inline papers/whitepaper.md:72]: The development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work. This allows experts in the community to earn a continuous stream of revenue through their model utilization.

File: papers/whitepaper.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+
+This is especially important once we factor in possible risks of AI censorship. For example, Italy [banned](https://www.bbc.com/news/technology-65139406) the use of OpenAI's ChatGPT shortly after its release, due to user privacy concerns (even though it later lifted the ban). On the other hand, companies like OpenAI are self-censoring due to societal and political reasons. For example, OpenAI will not predict the next presidential election, etc. These use cases will likely create large predictions/content markets that big technology companies will not want to be involved in. The Atoma Network, on the other hand, can leverage open-source software AI models, at a lower-cost GPU cluster to perform inference for these types of use cases.
```
Comment (joehill7) [inline papers/whitepaper.md:74]: On the other hand, companies like OpenAI are self-censoring due to societal and political reasons i.e OpenAI will not predict the next presidential election.

File: papers/whitepaper.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+
+The token distribution will follow an initial period of inflation in order to subsidize the network's growth. This growth will be used to subsidize future node operators to acquire powerful AI hardware, to incentivize model creators to develop new AI models, specialized for applications of interest to the Atoma Network (such as financial market predictions, etc), as well as to incentivize developers to build applications that drive demand for the available compute resources available on the Atoma Network. After the network's inflationary period (which will last a few years), the token emission distribution will follow a slightly deflationary dynamics, through a combination of tail emissions and burning mechanisms. Together, both strategies will lead to a gradual decrease in the total supply of the TOMA token, contributing for its overall value increase (provided the demand for Atoma Network increases), while also allowing for the allocation of the very small portion of newly minted tokens to the protocol's treasury that can then be used for different purposes, which can include bringing new generation hardware to the network, social incentives through UBI, etc.
```
Comment (joehill7) [inline papers/whitepaper.md:78]: The token distribution will follow an initial period of inflation in order to subsidize the network's growth. This growth will be used to:
1. subsidize future node operators to acquire powerful AI hardware, 
2. incentivize model creators to develop new AI models, specialized for applications of interest to the Atoma Network (such as financial market predictions)
3. incentivize developers to build applications that drive demand for the available compute resources available on the Atoma Network. 

After the network's inflationary period (which will last a few years), the token emission distribution will follow a slightly deflationary pattern through a combination of tail emissions and burning mechanisms. Together, both strategies will lead to a gradual decrease in the total supply of the TOMA token, contributing for its overall value increase (provided the demand for Atoma Network increases), while also allowing for the allocation of the very small portion of newly minted tokens to the protocol's treasury. These newly minted tokens can then be used for different purposes including bringing new generation hardware to the network or social incentives through UBI.

File: papers/whitepaper.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+
+As a decentralized platform, Atoma Network will put forth combined efforts that will ultimately lead to a new era of use cases within decentralized applications through driving financial incentives for computing power to be partially decoupled from centralized cloud providers. Smart contracts will not only deploy new financial products, but also be able to deploy and orchestrate compute resources, powering new decentralized protocols.
```
Comment (joehill7) [inline papers/whitepaper.md:80]: As a decentralized platform, Atoma Network will put forth combined efforts that will ultimately lead to a new era of use cases within decentralized applications. This is done so by driving financial incentives for computing power to be partially decoupled from centralized cloud providers. Smart contracts will not only deploy new financial products, but also be able to deploy and orchestrate compute resources, powering new decentralized protocols.

**Review by joehill7 — CHANGES_REQUESTED** [review]
Grammar and rewording changes

File: papers/whitepaper.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+
+For the above reasons, the Atoma Network is well positioned to lead the space of decentralized AI compute. We see a future where the Atoma Network will open the doors for a more bright, transparent, innovative, and democratic future where technology is fully aligned with user and community needs.
+
+### Introduction
+
+The past few years have seen the emergence of new generative AI capabilities powered by large language models (LLMs), more powerful hardware models which allow for massive scaling computing power, and a rise of highly scalable blockchains with lower transaction fees that utilize a combination of novel consensus, cryptography, and modularity techniques.
+
+To put this into perspective, McKinsey estimates that generative AI could add the equivalent of between $2.6 trillion to $4.4 trillion annually across the 63 use cases they analyzed, this would increase the impact of all artificial intelligence by 15% to 40%. By comparison, the United Kingdom’s entire GDP in 2021 was $3.1 trillion.
```
Comment (joehill7) [inline papers/whitepaper.md:35]: What does increase the impact mean more specifically? 

I.e the adoption of AI into businesses / our lives ?

**Pull request merged**